roles/o11y_agent/templates/agent-deb.yaml.j2

```diff
@@ -0,0 +1,203 @@
+server:
+  log_level: info
+
+{% if o11y_prometheus_url|length > 0 %}
+metrics:
+  wal_directory: {{ o11y_grafana_agent_wal_dir }}
+  global:
+    scrape_interval: 30s
+    external_labels:
+      host: {{ ansible_hostname }}
+{% if group_names|intersect(['maas_region_controller', 'maas_rack_controller']) %}
+      maas_az: {{ maas_availability_zone | default('default') }}
+      maas_region: {{ 'True' if 'maas_region_controller' in group_names else 'False' }}
+      maas_rack: {{ 'True' if 'maas_rack_controller' in group_names else 'False' }}
```

Comment:
Wouldn't this be the same as 
```jinja
maas_region: {{ 'maas_region_controller' in group_names }}
maas_rack: {{ 'maas_rack_controller' in group_names }}
```

Reply:
Yes, I think so-- thanks! It's a copy of the old o11y agent template, so what I'll do is create a separate PR updating all templates after this is merged-- seems cleaner to me.